zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/ParagraphRuntimeInfos.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.zeppelin.notebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 
+ *
+ */
+public class ParagraphRuntimeInfos {
+
+  String propertyName;
+  String label;
+  String group;
```

Comment:
I think here the `label` and `group` are duplicated. I don't see you use label in frontend.  Another approach I think of is that we always use interpreter group name as the prefix of `propertyName` as convention, (e.g. `spark.jobUrl`) @Leemoonsoo